src/uplift/bot/uplift_bot/bugzilla.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from libmozdata.bugzilla import Bugzilla
```

Comment:
Don't you need the latest version of libmozdata ? Uplift bot uses 0.1.40 right now

Reply:
I do. Can we upgrade the version Uplift bot uses?

Reply:
Awesome, thanks!